llama_index/output_parsers/selection.py

```diff
@@ -82,8 +82,13 @@ def _format_output(self, output: List[dict]) -> List[dict]:
         return output_json
 
     def parse(self, output: str) -> Any:
-        output = _marshal_llm_to_json(output)
-        json_output = json.loads(output)
+        json_output = _marshal_llm_to_json(output)
+        try:
+            json_output = json.loads(json_output)
+        except json.decoder.JSONDecodeError as exc:
+            raise ValueError(
+                f"Failed to convert LLM output {output!r} to JSON."
```

Comment:
huh, first I've seen this syntax -- is this supported by python3.8?

Reply:
Yeah it's a fun shorthand for `repr`: https://stackoverflow.com/questions/44800801/in-python-format-f-string-strings-what-does-r-mean

There is also `!s`, fwiw

Reply:
Oh and yeah it works in 3.8